matlab/GLMestimatesingletrial.m

```diff
@@ -1,123 +1,145 @@
 function results = GLMestimatesingletrial(design,data,stimdur,tr,outputdir,opt)
-
-% function results = GLMestimatesingletrial(design,data,stimdur,tr,outputdir,opt)
+%
+% USAGE::
+%
+%   results = GLMestimatesingletrial(design,data,stimdur,tr,outputdir,opt)
```

Comment:
the double `::` + lineskip + indent 

is what sphinx uses to render code.

The same way that in markdown:

```matlab
results = GLMestimatesingletrial(design,data,stimdur,tr,outputdir,opt)
```

is made with this

~~~
```matlab
results = GLMestimatesingletrial(design,data,stimdur,tr,outputdir,opt)
```
~~~